geode-core/src/main/java/org/apache/geode/distributed/internal/membership/gms/fd/GMSHealthMonitor.java

```diff
@@ -1277,6 +1277,9 @@ private boolean inlineCheckIfAvailable(final InternalDistributedMember initiator
     boolean failed = false;
 
     membersInFinalCheck.add(mbr);
+    suspectedMemberInView.putIfAbsent(mbr, this.currentView);
```

Comment:
@bschuchardt CHM.checkMember(run method) resets the next neighbor if its pinged. That means this method gets "not pinged"? 
But then view creator thread sees that it has been pinged?  But in this case, viewCreator thread should send view again? May we should walk this code.